From 0ec9341349971bea3cac44cf9854a71c50812c88 Mon Sep 17 00:00:00 2001
From: mickhuang <mickhuang@ami.com>
Date: Wed, 24 Jul 2024 11:50:33 +0800
Subject: [PATCH] Adjust dcscm flash layout

Since dcscm flash layout is too small and we temporarily dont need pfr functionally, so use evb layout instead it
---
 arch/arm64/boot/dts/aspeed/ast2700-dcscm.dts | 62 +-------------------
 1 file changed, 1 insertion(+), 61 deletions(-)

diff --git a/arch/arm64/boot/dts/aspeed/ast2700-dcscm.dts b/arch/arm64/boot/dts/aspeed/ast2700-dcscm.dts
index 91b5bb55884e..dee60a1ddba6 100644
--- a/arch/arm64/boot/dts/aspeed/ast2700-dcscm.dts
+++ b/arch/arm64/boot/dts/aspeed/ast2700-dcscm.dts
@@ -262,67 +262,7 @@ flash@0 {
 		spi-max-frequency = <12500000>;
 		spi-tx-bus-width = <4>;
 		spi-rx-bus-width = <4>;
-		partitions {
-			compatible = "fixed-partitions";
-			#address-cells = <1>;
-			#size-cells = <1>;
-			u-boot@0 {
-				reg = <0x0 0x200000>; // 2MB
-				label = "u-boot";
-			};
-			u-boot-env@200000 {
-				reg = <0x200000 0x20000>; // 128KB
-				label = "u-boot-env";
-			};
-			kernel@220000 {
-				reg = <0x220000 0x900000>; // 9MB
-				label = "kernel";
-			};
-			rofs@960000 {
-				reg = <0xb20000 0x24a0000>; // 36.625MB
-				label = "rofs";
-			};
-			rwfs@2e00000 {
-				reg = <0x2fc0000 0x1040000>; // 16.25MB
-				label = "rwfs";
-			};
-			pfm@4000000 {
-				reg = <0x4000000 0x20000>; // 128KB
-				label = "pfm";
-			};
-			reserved-1@4020000 {
-				reg = <0x4020000 0x200000>; // 128KB
-				label = "reserved-1";
-			};
-			rc-image@4220000 {
-				reg = <0x4220000 0x3de0000>; // 63360KB
-				label = "rc-image";
-			};
-			image-stg@8000000 {
-				reg = <0x8000000 0x3de0000>; // 63360KB
-				label = "img-stg";
-			};
-			pfr-stg@bde0000 {
-				reg = <0xbde0000 0x100000>; // 1024KB
-				label = "pfr-stg";
-			};
-			cpld-stg@bee0000 {
-				reg = <0xbee0000 0x400000>; // 4096KB
-				label = "cpld-stg";
-			};
-			afm-stg@c2e0000 {
-				reg = <0xc2e0000 0x20000>; // 128KB
-				label = "afm-stg";
-			};
-			afm-rc@c300000 {
-				reg = <0xc300000 0x20000>; // 128KB
-				label = "afm-rc";
-			};
-			reserved-2@c320000 {
-				reg = <0xc320000 0x3ce0000>; // 62336KB
-				label = "reserved-2";
-			};
-		};
+#include "aspeed-flash-layout-128.dtsi"
 	};
 };
 
-- 
2.34.1

